server/plugin.go

```diff
@@ -21,19 +21,19 @@ const (
 // ListManager representes the logic on the lists
 type ListManager interface {
 	// AddIssue adds a todo to userID's myList with the message
-	AddIssue(userID, message string) error
+	AddIssue(userID, message, postID string) error
```

Comment:
When postID added to the issue struct, let's pass and consume the `issue` object in methods.